### DISCUSSION TITLE
Integration Not Showing UP

### DISCUSSION THREAD

**Discussion by robhix** [gillesvs/librelink#22] [Q&A]
Followed the install instructions, but when I go to Settings->Devices & Services->Integrations and search for Librelink, nothing shows up. Yes, I did restart Home Assistant as specified.

HA Core: 2024.5.5
Supervisor: 2024.06.02
Operating System 12.3
Frontend: 20240501.1

**Comment by gillesvs** [comment]
Hello,
I think that your link to the repository is not good.
